app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,4 @@
+nav
+  ul.pager
```

Comment:
![screen shot 2016-01-22 at 10 41 49 am](https://cloud.githubusercontent.com/assets/4983239/12501127/c9a9e2c6-c0f4-11e5-9a2b-fb48adc6b5b1.png)

On the left is `.pager` and right is `.pagination`